nncf/tensorflow/graph/model_transformer.py

```diff
@@ -46,11 +65,55 @@ def __init__(self, model):
                 'Only tf.keras sequential or functional models can be transformed.')
 
         super().__init__(model)
+        self._converter = TFModelConverterFactory.create(model)
+        self._original_graph = self._converter.convert()
```

Comment:
ModelTransformer should not depend on NNCFGraph.

Reply:
It has to maintain an association between a node name and the underlying layer (layer name, instance idx, possibly more TF-specific info in the future that defines an object in the TF model to act upon) in order to parse the operations defined in terms of NNCFGraph. There is no way to do so without an NNCFGraph, or at least without NNCFNodeName.

Reply:
Removed graph generation from model transformer, switched back to storing TF-specific modification info in TF-target points generated by the TF subclass of the compression algo builder.